tests/unit/test_cassette_patterns.py

```
@@ -0,0 +1,434 @@
+
+def test_is_clean_flags_sid_starting_with_S_in_json_key() -> None:
+    """Same I7 hole in the JSON-dict-with-cookie-name-as-key shape."""
+    text = '{"SAPISID": "S_real_leaked_token_here"}'
+    ok, leaks = is_clean(text)
+    assert not ok
+
+
+def test_is_clean_flags_short_one_char_cookie_value() -> None:
+    """A single-character non-scrubbed leak (``"SID": "x"``) is detected."""
+    text = '{"SID": "x"}'
+    ok, leaks = is_clean(text)
```

Ruff (0.15.12)

[warning] Unpacked variable `leaks` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)